### ISSUE TITLE
Automatic login after tenant signup

### ISSUE THREAD

**Issue by riggraz** [astuto/astuto#263]
- [ ] When tenant signs up with OAuth, automatically log them in
- [ ] When tenant signs up with email, after clicking the confirmation link automatically log them in

Does it make sense? Not every website automatically logs you in when you confirm email address? Is there a reason for this?

**Comment by Brugman** [comment]
Makes sense. Email verification is a manual process and users can lie. You need the click. OAuth jumps to the remote service on it's own.
